lighthouse-core/computed/metrics/cumulative-layout-shift-all-frames.js

```diff
@@ -5,10 +5,38 @@
  */
 'use strict';
 
+/** @typedef {Omit<LH.TraceEvent, 'name'|'args'> & {name: 'LayoutShift', args: {data: {score: number, weighted_score_delta: number, had_recent_input: boolean}}}} LayoutShiftEvent */
+
 const makeComputedArtifact = require('../computed-artifact.js');
 const TraceOfTab = require('../trace-of-tab.js');
+const LHError = require('../../lib/lh-error.js');
 
 class CumulativeLayoutShiftAllFrames {
+  /**
+   * @param {LH.TraceEvent} event
+   * @return {event is LayoutShiftEvent}
+   */
+  static isLayoutShiftEvent(event) {
```

Comment:
nit: `isValidLayoutShiftEvent` or something? I wouldn't expect all of the validation and throw logic in a simple check

Reply:
I'm leaving this as is because the validation and throw logic are removed.